### PR TITLE
Improve CI config

### PR DESCRIPTION
**Pull request by szepeviktor**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -31,9 +31,6 @@ jobs:
         name: P${{ matrix.php }} - L${{ matrix.laravel }} - ${{ matrix.dependency-version }}
 
         steps:
-            - name: Update apt
-              run: sudo apt-get update --fix-missing
-
```
Comment (szepeviktor) [inline .github/workflows/tests.yml:36]: This is in vain.
Actually this abuses GitHub Actions 😢

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -47,6 +44,7 @@ jobs:
             - name: Install dependencies
               run: |
                   composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
-                  composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction --no-suggest
```
Comment (szepeviktor) [inline .github/workflows/tests.yml:50]: `--no-suggest` is deprecated

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -47,6 +44,7 @@ jobs:
...
+                  composer update --${{ matrix.dependency-version }} --prefer-dist --no-interaction
+
             - name: Execute tests
-              run: vendor/bin/phpunit
+              run: composer test
```
Comment (szepeviktor) [inline .github/workflows/tests.yml:50]: Be consistent by always using Composer.

**Comment by cesargb** [comment]
Thanks for your help!

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**